pandas/io/spss.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+from typing import Optional, Sequence, Union
+
+from pandas.core.api import DataFrame
+
+
+def read_spss(path: Union[str, Path],
+              usecols: Optional[Union[str, Sequence[str]]] = None,
+              categorical: bool = True) -> DataFrame:
```

Comment:
can we can call this ``convert_categoricals`` (similar to read_stata)

Reply:
Done.